src/atomate2/vasp/flows/hiphive.py

```diff
@@ -0,0 +1,527 @@
+        logger.info(f"loop = {loops}")
+        error_check_job = quality_control(
+            rmse_test=fit_force_constant.output[5],
+            n_structures=n_structures,
+            rattle_std=rattle_std,
+            loop=loops,
+            fit_method=fit_method,
+            disp_cut=disp_cut,
+            bulk_modulus=bulk_modulus,
+            temperature_qha=temperature_qha,
+            mesh_density=mesh_density,
+            imaginary_tol=imaginary_tol,
+            prev_dir_json_saver=json_saver.output[3],
+            prev_vasp_dir=prev_vasp_dir,
+            supercell_matrix_kwargs=supercell_matrix_kwargs,
+        )
+        error_check_job.name += f" {loops}"
+        jobs.append(error_check_job)
+        outputs.append(error_check_job.output)
+        error_check_job.metadata.update(
+            {
+                "tag": [
+                    f"error_check_job_{loops}",
+                    f"nConfigsPerStd={n_structures}",
+                    f"rattleStds={rattle_std}",
+                    f"dispCut={disp_cut}",
+                    f"supercell_matrix_kwargs={supercell_matrix_kwargs}",
+                    f"loop={loops}",
+                ]
+            }
+        )
+
+        # 7. Perform phonon renormalization to obtain temperature-dependent
+        # force constants using hiPhive
+        if renormalize:
+            for temperature in renormalize_temperature:
+                nconfig = renormalize_nconfig * (1 + temperature // 100)
+                renormalization = run_hiphive_renormalization(
+                    temperature=temperature,
+                    renorm_method=renormalize_method,
+                    nconfig=nconfig,
+                    conv_thresh=renormalize_conv_thresh,
+                    max_iter=renormalize_max_iter,
+                    renorm_TE_iter=renormalize_thermal_expansion_iter,
+                    bulk_modulus=bulk_modulus,
+                    mesh_density=mesh_density,
+                    prev_dir_hiphive=fit_force_constant.output[4],
+                    loop=loops,
+                )
+                renormalization.name += f" {temperature} {loops}"
+                jobs.append(renormalization)
+                outputs.append(renormalization.output)
+                renormalization.metadata.update(
+                    {
+                        "tag": [
+                            f"run_renormalization_{loops}",
+                            f"nConfigsPerStd={n_structures}",
+                            f"rattleStds={rattle_std}",
+                            f"dispCut={disp_cut}",
+                            f"supercell_matrix_kwargs={supercell_matrix_kwargs}",
+                            f"loop={loops}",
+                        ]
+                    }
+                )
+
+        # ##### 8. Lattice thermal conductivity calculation using phono3py
+        # if calculate_lattice_thermal_conductivity:
```

Comment:
Ah, okay, it's not included. Then, maybe, the command should be removed. Otherwise, I would recommend using the API of phono3py.